pyiron/atomistics/structure/atoms.py

```diff
@@ -2955,10 +2910,20 @@ def rotate_euler(self, center=(0, 0, 0), phi=0.0, theta=0.0, psi=0.0):
         rcoords = np.dot(a, np.transpose(rcoords))
         # Move back to the rotation point
         if self._is_scaled:
-            self.scaled_positions = np.transpose(rcoords) + center
+            self.get_scaled_positions(np.transpose(rcoords) + center)
```

Comment:
Should not this be set_scaled_positions ?

Reply:
brilliant comment -> I'm gonna fix it

Reply:
so this was not just a problem of this line. This function anyway doesn't seem to work. I'm looking into it more in detail